CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Optimizely Java X SDK Changelog
 
+## 3.2.1
+August 26th, 2019
```

Comment:
This date is 10 days into the future. I am assuming you meant 16th.

Reply:
🤦‍♂